beacon_api/conf/config.py

```diff
@@ -31,6 +30,8 @@
                                      user=os.environ.get('DATABASE_USER', 'beacon'),
                                      password=os.environ.get('DATABASE_PASSWORD', 'beacon'),
                                      database=os.environ.get('DATABASE_NAME', 'beacondb'),
+                                     # Multiple schemas can be used, and they need to be comma separated
+                                     server_settings={'search_path': DB_SCHEMA if DB_SCHEMA else 'public'},
```

Comment:
Just a note: we need both our schema name (`beacon`) and `public` in the `search_path`:
`export DATABASE_SCHEMA="beacon,public"`
If that's true for all cases, it might be helpful to append `public` per default here.

Reply:
ok, but I don't think we should default it, as the users might have different DB setups and some restrictions on the users for certain schemas. So making the administrators input it manually should be the option to go.